### PR TITLE
🌐 Add Korean translation for Tutorial - JSON Compatible Encoder

### PR DESCRIPTION
**Pull request by NEONKID**
Hello there.

Translated by referring to the previous PR. If you have any problems such as mistranslations, please feel free to leave your comments here.

This PR translates ```tutorial/encoder.md``` in Korean.

related: #2017

### CODE REVIEW

File: docs/ko/docs/tutorial/encoder.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{!../../../docs_src/encoder/tutorial001.py!}
+```
+
+이 예시는 Pydantic 모델을 `dict`로, `datetime` 형식을 `str`로 변환합니다.
+
+이렇게 호출한 결과는 파이썬 표준인 <a href="https://docs.python.org/3/library/json.html#json.dumps" class="external-link" target="_blank">`json.dumps()`</a>로 인코딩 할 수 있습니다.
+
+사이즈가 큰 문자열 형태의 JSON 형식(문자열)의 데이터가 들어있을 땐 `str`로 반환하지 않습니다. JSON과 모두 호환되는 값과 하위 값이 있는 Python 표준 데이터 구조 (예: `dict`)를 반환합니다.
+
+!!! note
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/encoder.md:33]:
```suggestion
!!! note "참고"
```

File: docs/ko/docs/tutorial/encoder.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+```Python hl_lines="5  22"
+{!../../../docs_src/encoder/tutorial001.py!}
+```
+
+이 예시는 Pydantic 모델을 `dict`로, `datetime` 형식을 `str`로 변환합니다.
+
+이렇게 호출한 결과는 파이썬 표준인 <a href="https://docs.python.org/3/library/json.html#json.dumps" class="external-link" target="_blank">`json.dumps()`</a>로 인코딩 할 수 있습니다.
+
+사이즈가 큰 문자열 형태의 JSON 형식(문자열)의 데이터가 들어있을 땐 `str`로 반환하지 않습니다. JSON과 모두 호환되는 값과 하위 값이 있는 Python 표준 데이터 구조 (예: `dict`)를 반환합니다.
```
Comment (peebbv6364) [inline docs/ko/docs/tutorial/encoder.md:31]:
```suggestion
길이가 긴 문자열 형태의 JSON 형식(문자열)의 데이터가 들어있는 상황에서는 `str`로 반환하지 않습니다. JSON과 모두 호환되는 값과 하위 값이 있는 Python 표준 데이터 구조 (예: `dict`)를 반환합니다.
```

File: docs/ko/docs/tutorial/encoder.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+예를 들면, `datetime` 객체는 JSON과 호환되는 데이터가 아니므로 이 데이터는 받지 않습니다.
+
+따라서 `datetime` 객체는 <a href="https://en.wikipedia.org/wiki/ISO_8601" class="external-link" target="_blank">ISO format</a> 데이터를 포함하는 `str`로 변환되어야 합니다.
+
+같은 방식으로 이 데이터베이스는 Pydantic 모델(속성이 있는 객체)을 받지 않고, `dict` 만을 받습니다.
+
+이를 위해 `jsonable_encoder` 를 사용할 수 있습니다.
+
+Pydantic 모델과 같은 객체를 받고 JSON 호환 가능한 데이터로 반환합니다:
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/encoder.md:21]:
```suggestion
Pydantic 모델과 같은 객체를 받고 JSON 호환 가능한 버전으로 반환합니다:
```
It's not a big deal; the original sentence used the term version, not data however.

It's good to keep the intention of the author as you can possible.

File: docs/ko/docs/tutorial/encoder.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+이 예시는 Pydantic 모델을 `dict`로, `datetime` 형식을 `str`로 변환합니다.
+
+이렇게 호출한 결과는 파이썬 표준인 <a href="https://docs.python.org/3/library/json.html#json.dumps" class="external-link" target="_blank">`json.dumps()`</a>로 인코딩 할 수 있습니다.
+
+사이즈가 큰 문자열 형태의 JSON 형식(문자열)의 데이터가 들어있을 땐 `str`로 반환하지 않습니다. JSON과 모두 호환되는 값과 하위 값이 있는 Python 표준 데이터 구조 (예: `dict`)를 반환합니다.
+
+!!! note
+`jsonable_encoder` 은 실제로 **FastAPI** 에서 내부적으로 데이터를 변환하는 데 사용하지만, 다른 많은 곳에서도 이는 유용합니다.
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/encoder.md:34]: Again, for preserving the intention, don't break a sentence structure, please. 🤗

And I think "는" is proper than "은" where `jsonable_encoder` after.

Comment (peebbv6364) [inline docs/ko/docs/tutorial/encoder.md:34]:
```suggestion
실제로 `jsonable_encoder`는 **FastAPI** 에서 내부적으로 데이터를 변환하는 데 사용되지만, 다른 많은 곳에서도 유용하게 사용됩니다.
```

File: docs/ko/docs/tutorial/encoder.md (hunk 1 above)
Comment (peebbv6364) [inline docs/ko/docs/tutorial/encoder.md:33]:
```suggestion
!!! 참고
```

File: docs/ko/docs/tutorial/encoder.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+데이터 유형(예: Pydantic 모델)을 JSON과 호환된 형태로 반환해야 하는 경우가 있습니다. (예: `dict`, `list` 등)
+
+예를 들면, 데이터베이스에 저장해야하는 경우입니다.
+
+이를 위해, **FastAPI** 에서는 `jsonable_encoder()` 함수를 제공합니다.
+
+## `jsonable_encoder` 사용
+
+JSON 호환 가능 데이터만 수신하는 `fake_db` 데이터베이스가 있다고 가정해보겠습니다.
```
Comment (peebbv6364) [inline docs/ko/docs/tutorial/encoder.md:11]:
```suggestion
JSON 호환 가능 데이터만 수신하는 `fake_db` 데이터베이스가 존재한다고 가정하겠습니다.
```

File: docs/ko/docs/tutorial/encoder.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+예를 들면, `datetime` 객체는 JSON과 호환되는 데이터가 아니므로 이 데이터는 받지 않습니다.
```
Comment (peebbv6364) [inline docs/ko/docs/tutorial/encoder.md:13]:
```suggestion
예를 들면, `datetime` 객체는 JSON과 호환되는 데이터가 아니므로 이 데이터는 받아들여지지 않습니다.
```

**Review by peebbv6364 — COMMENTED** [review]
I propose some modifications. However, you can reject the offer because it is simply an expression of my thoughts.

File: docs/ko/docs/tutorial/encoder.md (hunk 1 above)
Comment (hard-coders) [reply docs/ko/docs/tutorial/encoder.md:33]: We have to write like:
```suggestion
!!! note "참고"
```

**Comment by NEONKID** [comment]
@hard-coders @peebbv6364 
Thank you for reviewing. The opinions you gave were actively accepted and reflected.

**Review by hard-coders — APPROVED** [review]

**Comment by 0417taehyun** [comment]
@peebbv6364

Can you approve the review?

**Review by peebbv6364 — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks for your contribution @NEONKID! :cake: 

And thanks for the reviews @hard-coders and @peebbv6364! :bow:

**Pull request merged**